README.md

```diff
@@ -169,6 +169,21 @@ Next, provide the path to the `subnet-evm` repository and the path to a writeabl
 ./scripts/local/e2e_test.sh --local --subnet-evm ./subnet-evm --data-dir ~/tmp/e2e-test
 ```
 
+#### Run the E2E tests on another network
+
+The E2E tests can be run on another network by implementing the `Network` interface in `package network`. For example, the type `FujiNetwork` in `example_fuji_network.go` implements this interface, pointing to the Amplify, Bulletin, and Conduit Fuji subnets. After implementing this interface, you can run the E2E tests on this network by running a program such as:
```

Comment:
might be worth linking where to find the fuji ABC subnet explorers

Reply:
Good call. Added. Though we will need to make sure to update this when these subnets are replaced.